application/akka/java/example/src/main/java/com/example/Example.java

```diff
@@ -0,0 +1,78 @@
+package com.example;
+
+import akka.NotUsed;
+import akka.actor.typed.Behavior;
+import akka.http.javadsl.Http;
+import akka.http.javadsl.ServerBinding;
+import akka.http.javadsl.server.Route;
+import akka.actor.typed.javadsl.Behaviors;
+import akka.actor.typed.ActorSystem;
+import akka.http.javadsl.server.AllDirectives;
+
+import java.net.InetSocketAddress;
+import java.util.concurrent.CompletionStage;
+
+//#main-class
```

Comment:
I'm also not a fan of these comments, but if they are autogenerated and you think users are used to them, then they are fine by me.

Reply:
They are made by the template. Also, @florisvdg suggested to remove them, since they don't bring any value. 
Also, I looked again on their guides and such comments are not present, therefore they will be removed.